src/components/ChatEntry.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const chatClass =
+    props.sender === 'Estragon' ? 'chat-entry remote' : 'chat-entry local';
```

Comment:
Since this hard codes "Estragon" it's a little brittle.  The code would break if we wanted to render a different chat between two other people.